rosbag2_to_video/api.py

```diff
@@ -30,25 +30,45 @@ def get_stamp_from_image_msg(image_msg):
 
 def add_arguments_to_parser(argparser):
     argparser.add_argument(
-        'bagfile', help='Path to the bag (currently only supports sensor_msgs/msg/Image types)')
-    argparser.add_argument('-t','--topic', required=True, help='Name of the image topic')
-    argparser.add_argument('-o','--output', required=True, help='Output filename (currently only supports MP4)')
+        'bagfile',
+        help='Path to the bag (currently only supports sensor_msgs/msg/Image or sensor_msgs/msg/CompressedImage types)' # noqa E501
```

Comment:
could you move the comment about the supported types to `-t/--topic`?
Thanks!

Reply:
@ivanpauno I moved the comment in https://github.com/ivanpauno/rosbag2_to_video/pull/1/commits/741b1e325878d7a65f413fba7193859bf5646778.